src/matrixrtc/CallMembership.ts

```diff
@@ -184,10 +196,24 @@ export class CallMembership {
         }
     }
 
+    /**
+     * @returns The number of milliseconds until the membership expires or undefined if applicable
+     */
     public getMsUntilExpiry(): number | undefined {
-        if (isLegacyCallMembershipData(this.membershipData)) return this.getLocalExpiry()! - Date.now();
+        if (isLegacyCallMembershipData(this.membershipData)) {
+            // Assume that local clock is sufficiently in sync with other clocks in the distributed system.
```

Comment:
Perhaps this is notable enough to appear in the docstring...but since it refers to & relies on internal behaviour / non-exported helper values & functions (notably `MatrixRTCSession.membershipEventNeedsUpdate`), it's more of a notice to js-sdk developers than for people using this as a dependency.  So this is fine as is.